### ISSUE TITLE
Images in resource file

### ISSUE THREAD

**Issue by dschick** [caqtdm/caqtdm#43]
I was trying to add a logo image using a rich text label.
I created a resource file and addes an image, which is located in the same folder as my .ui files:
- root/panel.ui
- root/logo.png

In the `panel.ui` file I get the link to the image as:
`:/logo.png`
which seems correct.
However, caQtDM is not displaying the image.
If I manually change the .ui file to
`./logo.png`
it works.

What is my mistake here?

Best

Daniel

**Comment by dschick** [comment]
thanks @HelgeBrands for the quick reply.
I was either starting caQtDM from within the folder of the UIs and also tested with CAQTDM_DISPLAY_PATH set to that folder.
There are actually no error messages during start-up of caQtDM.

my resource file within the same root folder looks like:
```
<RCC>
  <qresource>
    <file>logo.png</file>
  </qresource>
</RCC>
```

**Comment by HelgeBrands** [comment]
sorry now I'm a little bit confused ;-) 
As I know the RCC file needs to be compiled. But when you are "only" using caQtDM than this can't work, because the UI file is parsed during run time and not compiled to a CPP file. You can try this with one of our RCC files 
e.g. :/caQtDM-logospsi.png 
Your logo.png can be added during compile time, but not in an RCC file. Here you need the filesystem.

**Comment by dschick** [comment]
ah okay, sorry, I havn't worked with qt before.
So then I just set the source of the image to 
./logo.png
and do not work with the resource file but instead I have to use the CAQTDM_DISPLAY_PATH, to tell caQtDM where the ui- and image-files are located, right?

Best

Daniel

**Comment by HelgeBrands** [comment]
Yes, I would use a sub directory

**Comment by dschick** [comment]
hm, I just tried it today and the image is still not showing.
The strange thing is, that it is working on the machine where I am creating the .ui files with the designer.
On another machine (a raspi) the image is not showing.

I have set the CAQTDM_DISPALY_PATH on all systems.
It also seems to work, because I can open new ui widgets from a main widget (which does not work, if I start caQtDM from outside the CAQTDM_DISPALY_PATH and CAQTDM_DISPALY_PATH not set.

However, the image is not showing unless I change the Link in the label manually from a relative path:
./logo.png
to
"CAQTDM_DISPALY_PATH"/logo.png

anything else I might test?
